CONTRIBUTING.md

```diff
@@ -304,8 +304,73 @@ We are excited to help you contribute to any of the topics that you would like t
 
 When in doubt, you can reach out to current project lead(s):
 
-| Name            | GitHub | Twitter |
-|:----------------|:-------|:--------|
-| Quincy Larson | [`@QuincyLarson`](https://github.com/QuincyLarson) | [`@ossia`](https://twitter.com/ossia)|
+| Name            | GitHub | Role |
+|:----------------|:-------|:-----|
+| Fran Zeko | [@Zeko369](https://github.com/Zeko369) | Admin UI, routes, models, and data migrations
+| Ayotomide Oladipo | [@tomiiide](https://github.com/tomiiide) | Public-facing client pages / forms
+| Timmy Chen | [@timmyichen](https://github.com/timmyichen) | API
+| Patrick San Juan | [@pdotsani](https://github.com/pdotsani) | Google Authentication
+| Jonathan Seubert | [@megajon](https://github.com/megajon) | Email
+| Vaibhav Singh | [@vaibhavsingh97](https://github.com/vaibhavsingh97) | Heroku 1-click deployment
+| Jim Ciallella | [@allella](https://github.com/allella) | Documentation
+| Quincy Larson | [@QuincyLarson](https://github.com/QuincyLarson) | Executive Lead
+
+
 
 You are a champion :).
+
+
+# Server-side Technical Documentation
+
+## Database
+
+for any problems ping @Zeko369
+
+We're using Postgres for our database and TypeORM for our ORM (mapping database tables to js objects). Here are a few examples on how to use our TypeORM setup.
+
+To make it since the port from docker postgres service is exposed to the host (54320) you dont have to run `docker-compose exec...`, and can just run db commands from the host. This also makes it a lot easier to access the db from the ourside if you're running a local db on the system.
```

Comment:
```suggestion
To make it since the port from docker `postgres` service is exposed to the host (54320, not to mess with local `postgres` if you're running one) you don't have to run `docker-compose exec...`, and can just run db commands from the host. This also makes it a lot easier to access the db from the outside if you're not running a local db on the system to for example open it in [Postico](https://eggerapps.at/postico/) / [Table Plus](https://tableplus.com/).
```